lib/tasks/docker.rake

```diff
@@ -33,11 +35,18 @@ end
 
 desc 'Run JS and Ruby linters'
 task 'docker:lint' do
-  success = run_or_fail("bundle exec rubocop --parallel")
-  success = run_or_fail("eslint app/assets/javascripts test/javascripts")
-  success = run_or_fail("eslint --ext .es6 app/assets/javascripts test/javascripts plugins")
+  success = true
+
+  if ENV["SINGLE_PLUGIN"]
+    success &&= run_or_fail("bundle exec rubocop --parallel plugins/#{ENV["SINGLE_PLUGIN"]}")
+    success &&= run_or_fail("eslint --ext .es6 plugins/#{ENV['SINGLE_PLUGIN']}")
+  else
+    success &&= run_or_fail("bundle exec rubocop --parallel") unless ENV["SKIP_CORE"]
+    success &&= run_or_fail("eslint app/assets/javascripts test/javascripts") unless ENV["SKIP_CORE"]
+    success &&= run_or_fail("eslint --ext .es6 app/assets/javascripts test/javascripts plugins") unless ENV["SKIP_PLUGINS"]
+  end
 
-  exit 1 if !success
+  exit 1 unless success
```

Comment:
just a tiny one... I find unless SUPER confusing. in this case it is questionably ok, but it makes code require tons of mental gymnastics just to figure out what it does

open_door unless police_officer || clown

open_door if !policy_officer && !clown